<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlazeRenderer</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="GLWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Read"/>
    <addaction name="actionSave_screenshot"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action1D_TF"/>
    <addaction name="separator"/>
    <addaction name="actionRaycasting_settings"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Read"/>
   <addaction name="actionSave_screenshot"/>
   <addaction name="separator"/>
   <addaction name="actionRaycasting_settings"/>
   <addaction name="action1D_TF"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Read">
   <property name="icon">
    <iconset resource="../../res/icons.qrc">
     <normaloff>:/icons/open_volume.png</normaloff>:/icons/open_volume.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Read volume</string>
   </property>
   <property name="toolTip">
    <string>Read volume data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action1D_TF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/icons.qrc">
     <normaloff>:/icons/TF1D.png</normaloff>:/icons/TF1D.png</iconset>
   </property>
   <property name="text">
    <string>1D Transfer Function</string>
   </property>
   <property name="toolTip">
    <string>1D Transfer Function</string>
   </property>
  </action>
  <action name="actionRaycasting_settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/icons.qrc">
     <normaloff>:/icons/render_settings.png</normaloff>:/icons/render_settings.png</iconset>
   </property>
   <property name="text">
    <string>Raycasting settings</string>
   </property>
   <property name="toolTip">
    <string>Raycasting settings</string>
   </property>
  </action>
  <action name="actionSave_screenshot">
   <property name="icon">
    <iconset resource="../../res/icons.qrc">
     <normaloff>:/icons/screenshot.png</normaloff>:/icons/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Save screenshot</string>
   </property>
   <property name="toolTip">
    <string>Save screenshot of volume render</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>ui/glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
